plugins/woocommerce/src/Internal/ProductDownloads/ApprovedDirectories/Register.php

```diff
@@ -31,6 +31,9 @@ class Register {
 	 */
 	private $mode_option = 'wc_downloads_approved_directories_mode';
 
+	/** @var array<string, bool> cached valid paths */
+	private array $valid_paths = [];
```

Comment:
```suggestion
	private array $valid_paths = array();
```
WPCS prefers longform array declaration.